CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# v0.20.0
+## Changed
+* Update dependencies
+
+## Added
+* template_provider: Add example for using FsSingleTemplateProvider to get hash of a local tempalte set to README 
```

Comment:
"tempalte" -> "template"

However, I usually do not document README changes in the CHANGELOG.